src/functions/enclose.js

```diff
@@ -127,8 +129,13 @@ const mathmlBuilder = (group, options) => {
             node.setAttribute("notation", "box");
             break;
         case "\\fcolorbox":
-            // TODO(ron): I don't know any way to set the border color.
-            node.setAttribute("notation", "box");
+            // <menclose> cannot set a border color. So use <mpadded> instead.
+            // Set some attributes that come included with <menclose>.
+            node.setAttribute("width", "+10px");
+            node.setAttribute("height", "+5px");
+            node.setAttribute("lspace", "5px");
+            node.setAttribute("style", "border: 1.5px solid " +
```

Comment:
Are these values documented somewhere?

Reply:
I used the same values as MathJax. If you would prefer the LaTeX2e padding, I would have no objection.

I do object to the LaTeX2e border width. On my screen, I can barely see it. So for this border, I chose a 1.5px value as a happy medium between the too-thin LaTeX2e (0.04em) border and the too-gaudy MathJax (2px) border.